include/seqan3/range/views/translate_join.hpp

```diff
@@ -291,22 +291,26 @@ struct translate_join_fn
     template <std::ranges::range urng_t>
     constexpr auto operator()(urng_t && urange, translation_frames const tf = translation_frames::SIX_FRAME) const
     {
-        static_assert(dimension_v<urng_t> == 2,
-            "This adaptor only handles range-of-range (two dimensions) as input.");
+        static_assert(range_dimension_v<urng_t> == 2,
+                     "This adaptor only handles range-of-range (two dimensions) as input.");
```

Comment:
```suggestion
                      "This adaptor only handles range-of-range (two dimensions) as input.");
```

one more :see_no_evil: sorry

Reply:
thanks for spotting ! 😁